tests/Extractor/HttpClientTest.php

```diff
@@ -568,39 +327,21 @@ public function testTimeout()
 
     public function testNbRedirectsReached()
     {
-        $response = $this->getMockBuilder('GuzzleHttp\Message\Response')
-            ->disableOriginalConstructor()
-            ->getMock();
-
-        $response->expects($this->any())
-            ->method('getEffectiveUrl')
-            ->willReturn('http://fr.wikipedia.org/wiki/Copyright');
-
-        $response->expects($this->any())
-            ->method('getHeaders')
-            ->willReturn([]);
-
-        $response->expects($this->any())
-            ->method('getStatusCode')
-            ->willReturn(200);
-
-        $response->expects($this->any())
-            ->method('getBody')
-            ->willReturn('<meta HTTP-EQUIV="REFRESH" content="0; url=http://fr.wikipedia.org/wiki/Copyright?' . rand() . '">');
-
-        $client = $this->getMockBuilder('GuzzleHttp\Client')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $httpMockClient = new HttpMockClient();
 
-        $client->expects($this->any())
-            ->method('get')
-            ->willReturn($response);
+        for ($i = 0; $i < 10; $i++) {
```

Comment:
The configuration of redirection by httplug was incorrect and the test was not well implemented, I changed it. Is it better ?